qml/DeviceManagerViewer.qml

```diff
@@ -209,7 +209,7 @@ PingPopup {
                         Label {
                             Layout.fillWidth: true
                             horizontalAlignment: Text.AlignHCenter
-                            text: connection.typeToString() + " " + connection.createConfString()
+                            text: connection.typeToString() + " " + connection.argsAsConst()[0]
```

Comment:
what happens if argsAsConst() is empty?

Reply:
This is not possible, connection is a property that represents the link configuration of a specific sensor. If connection exists in devicemanager, it was a search result of the protocol detection logic and someone has answered it. It's not possible to communicate with a sensor without a valid connection structure.